### ISSUE TITLE
Login from external ip using port forwarding 

### ISSUE THREAD

**Issue by devenashani** [SiCKRAGETV/sickrage-issues#2193]
Python Version: 2.7.3 (default, Jul 13 2012, 04:03:47) [GCC 4.4.5]
Operating System: Linux-2.6.31.8.nv+v2-armv5tel-with-debian-6.0.3
Branch: master
Commit: ea897589698c18007ff0fd062b224d68d76d3bb5
Link to Log: https://gist.github.com/devenashani/95a025ea865d38d80849

Problem Faced: 

I have enabled port forwarding on my router to point to my internal sickrage ip and port, every thing was working fine till couple of days back, now when I try to login to my sigrage GUI from outside my LAN network using public ip, I get login password screen, after putting in the username & password and clicking login, I am not able to go further, the same login & password screen appears again.

upon checking the log I get the below given error.

```
2015-07-18 14:45:48 WARNING  TORNADO :: User attempted a failed login to the SickRage web interface from IP: xxx.xxx.xxx.xxx
```

I have masked the ip for privacy. If i login by using local LAN ip everything works OK

Can you please look in the matter, as I some time login using my public ip from my office and getting back this will be great.

**Comment by neoatomic** [comment]
Please follow the guidelines for posting an issue. : 
https://github.com/SiCKRAGETV/sickrage-issues#submitting-a-bugissue-ticket

Also port forwarding is a very unsafe method to access your network. You can better look into setting up a VPN connection.

**Comment by devenashani** [comment]
I have updated the post.

Hope this will help. Please look into the issue.

As VPN passthrough is blocked in my office network, I have configured my router to accept remote connections from specific IP as a way around.

**Comment by neoatomic** [comment]
Cant recall that we did change anything to the login section. And sadly the log isn't helpful...
Are you using https.? Try to disable and try again.  Also try using another browser and see if that works.

**Comment by fernandog** [comment]
Stop SR
edit config file.
erase user/pass
save
try again.

**Comment by devenashani** [comment]
@neoatomic I am not using https

@fernandog Your TIP helped, i stopped SR cleared the username and password then started the SR again, I was not able to login. But i again stopped the SR, re-entered the username and password in config file, now i am able to login from the outside ip.. PROBLEM SOLVED !!! THanks for your help.

**Comment by 1DontEx1st** [comment]
I'm having this same issue, with 
SR Version:	BRANCH: (master) / COMMIT: (540169fb909f3fb8dec1f19af576a8f4e76ca8b9), however the tip from @fernandog that worked for @devenashani did *not* work for me

The warning message in the log is:
2015-10-09 10:25:45 ERROR    TORNADO :: User attempted a failed login to the SickRage web interface from IP: [redacted]

If I remove the u/p in the config I'm able to log in just fine via the forwarded port, but once I add it back I can no longer lot in remotely. I recently switched from an older version of SickBeard which I was able to access in the exact same way that I'm trying to access SickRage now, if that means anything.

Thanks!
